sdo/activity_sdo_context.jsonld

```diff
@@ -23,6 +23,38 @@
       "@type": "sdo:DateTime"
     },
     "location": "sdo:location",
-    "performedBy": "sdo:agent"
+    "performedBy": "sdo:agent",
+    "alternateIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "relatedIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "dates": {
+      "@id": "sdo:temporalCoverage",
+      "@type": "sdo:DateTime"
+    },
+    "duration": {
+      "@id": "sdo:activityDuration",
+      "@type": "sdo:Thing"
```

Comment:
@surchs do you think we can have sdo:Duration here? it's a subclass of Thing and basically inherits all the same properties. The benefit in changing it is that [sdo:Duration](https://schema.org/Duration) specifies that it uses ISO 8601 duration format.